object-manipulation-1/src/set-value.ts

```diff
@@ -0,0 +1,4 @@
+/* exported setValue */
+function setValue(object: object, key: string, value: any): any {
```

Comment:
Use return type of "void" for return type of functions that don't return any values...